ui/packages/consul-ui/tests/acceptance/dc/services/show/topology/stats.feature

```diff
@@ -33,4 +33,50 @@ Feature: dc / services / show / topology / stats
     ---
     And I see topology on the tabs
     And I see the "[data-test-topology-metrics-stats]" element
+  Scenario: Given metrics is enabled, all metrics are disabled for an ingress gateway Topology
+    Given 1 datacenter model with the value "dc1"
+    Given a "prometheus" metrics provider
+    And 1 node models
+    And 1 service model from yaml
+    ---
+    - Service:
+        Name: ingress-gateway
+        Kind: ingress-gateway
+        ID: ingress-gateway-with-id
+    ---
+    When I visit the service page for yaml
+    ---
+      dc: dc1
+      service: ingress-gateway
+    ---
+    And I see topology on the tabs
+    And I don't see the "[data-test-topology-metrics-stats]" element
+    And I see the "[data-test-sparkline]" element
```

Comment:
```gherkin
Given metrics is enabled, all metrics are disabled for an ingress gateway Topology
Then I see the "[data-test-sparkline]" element
```
This reads like the metrics/sparkline is still visible? Whereas, from what I understand, it should be hidden here?

Reply:
I've updated the scenario.